docs/gettinginvolved/functional-testing.rst

```diff
@@ -45,6 +52,12 @@ The configuration stanza for the OpenStack backend running on Rackspace looks as
      username: <rackspace username>
      key: <access key>
 
+To run the functional tests, run the following command:
```

Comment:
presumably its just removing the region field, as its set in the ``FLOCKER_FUNCTIONAL_TEST_OPENSTACK_REGION`` variable?